Extractor/Config.cs

```diff
@@ -642,7 +640,43 @@ public class SubscriptionConfig
         /// Log bad subscription datapoints
         /// </summary>
         public bool LogBadValues { get; set; } = true;
+
+        /// <summary>
+        /// Server namespace nodes the should be subscribed to a rebrowse upon changes to their values.
+        /// </summary>
+        public RebrowseTriggersConfig? RebrowseTriggers { get; set; }
```

Comment:
Thinking about it, I'm not so sure this belongs in `Subscriptions`, yes it does create subscriptions, but this section is generally more about modifying the data subscriptions we make. Things related to rebrowse are generally under `Extraction`.